### PR TITLE
[ADD] Inclusion of multiple answers to audio tests.

### PR DESCRIPTION
**Pull request by rogerioefonseca**
Why:
Sometimes is going to be possible to exist multiple answers for the
samne question.
Using this algorithm is going to be possible to work with Extended
numbes too.

Example:

    "path": "thirdmonth.mp3",
    "value": "march|3"

### CODE REVIEW

File: src/visualCaptcha/assets/audios.json [hunk 1]
```diff
@@ -76,6 +76,10 @@
         "value": "march"
```
Comment (BrunoBernardino) [inline src/visualCaptcha/assets/audios.json]: Shouldn't this line be changing instead of adding a new one?

File: src/visualCaptcha/Captcha.php [hunk 2]
```diff
@@ -187,8 +187,8 @@ public function validateImage( $sentOption ) {
     // Validate the sent audio value with the validAudioOption
     public function validateAudio( $sentOption ) {
         $validAudioOption = $this->getValidAudioOption();
-
-        return ( $sentOption == $validAudioOption[ 'value' ] );
+        $validAnwser = explode("|", strtoupper($validAudioOption[ 'value' ]));
```
Comment (BrunoBernardino) [inline src/visualCaptcha/Captcha.php]: Can you please rename `$validAnwser` to `$validAnswers` so it's clearer it's an array when used below? Also fixing the typo in its name.

File: src/visualCaptcha/assets/audios.json (hunk 1 above)
Comment (rogerioefonseca) [inline src/visualCaptcha/assets/audios.json]: You are right. 
I have done it!

File: src/visualCaptcha/Captcha.php (hunk 2 above)
Comment (rogerioefonseca) [inline src/visualCaptcha/Captcha.php]: Sure Bruno, renamed!!

**Comment by BrunoBernardino** [comment]
Awesome, thanks!

**Pull request merged**